Source/Plugin.BLE.iOS/BleImplementation.cs

```diff
@@ -10,11 +11,17 @@ namespace Plugin.BLE
 {
     internal class BleImplementation : BleImplementationBase
     {
+        private readonly CBCentralInitOptions _cbCentralInitOptions;
         private CBCentralManager _centralManager;
 
+        public BleImplementation(CBCentralInitOptions cbCentralInitOptions)
+        {
+            _cbCentralInitOptions = cbCentralInitOptions ?? new CBCentralInitOptions();
+        }
+
         protected override void InitializeNative()
         {
-            _centralManager = new CBCentralManager(DispatchQueue.MainQueue);
+            _centralManager = new CBCentralManager(null, DispatchQueue.MainQueue, _cbCentralInitOptions);
```

Comment:
Look at lifecycle for event. Can we use delegate instead

Reply:
@ThomasHoest  It seems a bit odd with the "BleImplementation" since, it listens to a event here, in order to update properties which are used in the app. So a delegate might not do a lot - but maybe making this into a singleton, or make a Finalize method would fix the event issue?